docs/data-ai/ai/run-inference.md

```diff
@@ -18,57 +18,64 @@ aliases:
 description: "Run inference on a model with a vision service or an SDK."
 ---
 
-After deploying an ml model, you need to configure an additional service to use the inferences the deployed model makes.
-You can run inference on an ML model with a vision service or use an SDK to further process inferences.
+Inference is the process of generating output from a machine learning (ML) model.
+With Viam, you can run inference to generate the following kinds of output:
 
-## Use a vision service
+- object detection (using bounding boxes)
+- classification (using tags)
 
-Vision services work to provide computer vision.
-They use an ML model and apply it to the stream of images from your camera.
+You can run inference locally on a Viam machine, or remotely in the Viam cloud.
 
-{{<resources_svc api="rdk:service:vision" type="vision">}}
-
-{{< readfile "/static/include/create-your-own-mr.md" >}}
-
-Note that many of these services have built in ML models, and thus do not need to be run alongside an ML model service.
+## Machine
 
-One vision service you can use to run inference on a camera stream if you have an ML model service configured is the `mlmodel` service.
+You can use `viam-server` to deploy and run ML models directly on your machines.
 
-### Configure an mlmodel vision service
+You can run inference on your machine in the following ways:
 
-Add the `vision / ML model` service to your machine.
-Then, from the **Select model** dropdown, select the name of the ML model service you configured when [deploying](/data-ai/ai/deploy/) your model (for example, `mlmodel-1`).
+- with a vision service
+- manually in application logic with an SDK
 
-**Save** your changes.
+Typical machines, like Raspberry Pi 4, can run small TensorFlow Lite (TFLite) models.
```

Comment:
I don't know if we want to call them "typical" or something that describes why they can / can't run certain models based on their spec

Reply:
Agreed. Rephrased both sentences and changed this to 'entry-level', which is still not strictly 100% perfectly true, but I think is near enough.